### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by anshuld1995** [community/community#20795] [Other Feature Feedback, Questions, & Ideas] [answered]
hello everyone.
i'am Data scientist i need a help to create a blog in some specific poject , so can anybody help me or give some ideas how to write and what content ,,,,,plz

**Comment by MathiasReker** [comment]
Check out this repo: https://github.com/osopromadze/Spring-Boot-Blog-REST-API

This is a good starter to create a REST API in Java. Then you can create a frontend in JS. :-)

**Reply by anshuld1995** [reply]
thankyou

**Comment by mohsin-riad** [comment, marked as answer]
as you work on data, write tutorials and show how you achieved something on specific project such as **Data Preprocessing, Data splitting, Missing Data manipulation, Data Scrapping, Data Visualization**, etc.  

Furthermore, if you worked on Machine learning algorithms, you can elaborate on how you split and scale the Test and Train data. 
you can check here how I did: 
- [ML-Documentation](https://github.com/mohsin-riad/neural-network/tree/main/Experiments/Texual-Emotion)
- [Automation-Tutorial](https://github.com/mohsin-riad/automation-bijoy-to-avro)

if you find my answer helpful, please feel free to `mark as answered`

**Reply by anshuld1995** [reply]
its help me a lot thankyou

**Reply by mohsin-riad** [reply]
great, You're welcome 😊
